### PR TITLE
Make sure to create objects in provided namespace

### PR DESCRIPTION
**Pull request by pkliczewski**
When we run apb we ask user to provide target namespace/project and
in the code we need to respect it.

This PR provides small fixes to the j2 files which caused issues during
oc create.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -49,6 +49,11 @@ COPY playbooks /opt/apb/actions
 COPY roles /opt/ansible/roles
 RUN chmod -R g=u /opt/{ansible,apb}
 RUN yum -y install libvirt-client curl qemu-img wget && yum clean all
+RUN wget https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
+RUN tar zxvf openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
```
Comment (aglitke) [inline Dockerfile]: Why is this new version needed?

File: roles/provision-v2v/tasks/login.yml [hunk 2]
```diff
@@ -4,3 +4,6 @@
 
 - name: Change project to {{ namespace }}
   command: "oc project {{ namespace }}"
+
+- name: Add priviliged policy for User
+  command: "oc adm policy add-scc-to-user privileged system:serviceaccount:{{ namespace }}:kubevirt-privileged"
```
Comment (aglitke) [inline roles/provision-v2v/tasks/login.yml:9]: What if this exists already?  Seems a common thing I've seen in playbooks?  Isn't it kubevirt's job to give its users appropriate permissions?

**Review by aglitke — COMMENTED** [review]
There are lots of fixes in here besides the ns fixes.  Please either split the commits or just enumerate the fixes you're doing in the commit message.

File: Dockerfile (hunk 1 above)
Comment (pkliczewski) [reply Dockerfile]: there is an issue with oc apply. We used to use 3.7 which is buggy. Now we will use 3.9.

File: roles/provision-v2v/tasks/login.yml (hunk 2 above)
Comment (pkliczewski) [reply roles/provision-v2v/tasks/login.yml:9]: Pod which is using `virt-v2v` needs to run as privileged container so we need the ability to create one. That is why we need it.

Comment (aglitke) [reply roles/provision-v2v/tasks/login.yml:9]: Ok.  so just run this pod as the kubevirt-privileged serviceaccount.

File: Dockerfile (hunk 1 above)
Comment (aglitke) [reply Dockerfile]: Please add a comment about this in the Dockerfile.

Comment (pkliczewski) [reply Dockerfile]: Done

File: roles/provision-v2v/tasks/login.yml (hunk 2 above)
Comment (pkliczewski) [reply roles/provision-v2v/tasks/login.yml:9]: Correct

**Review by aglitke — APPROVED** [review]

**Pull request merged**